.github/workflows/stale.yml

```diff
@@ -7,10 +7,11 @@ on:
 
 jobs:
   stale:
-    if: github.event.repository.fork == false
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/stale@v4
+    - run: echo "IS_FORK=$(curl "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY" | jq -r .fork)" >> $GITHUB_ENV
+    - if: env.IS_FORK == 'false'
```

Comment:
That is so fancy, why not just take known knowledge we have - that is, we *know* the canonical main repo - and use it for a clean check like https://github.com/invertase/firestore-ios-sdk-frameworks/blob/a10077acdf3e9d4fbd006a729d4dac45711c657c/.github/workflows/build.yml#L11 ?

Reply:
I agree; let's just keep it simple for now.

Reply:
Sure, a check like that would work as well.
What's nice about the current (and proposed) workflow is that it is completely repo agnostic - Meaning there is nothing specific to react-native-device-info in it, it can be copied and reused without changes in any other repo.
If you prefer to hard-code the repo org and name in the file instead, I can also do that - Let me know and I'll amend the commit. :+1:

Reply:
sorry this stalled, I like the idea of a snippet that's reusable across repos, that's a strong point and I'll probably end up copy/pasting this around myself.

Reply:
Thanks @mikehardy!